### ISSUE TITLE
Fail to build

### ISSUE THREAD

**Issue by mquinson** [schwering/irclib#9]
Hello,

we have a long-standing bug in the Debian system, where the robots indicate that this package does not build from source anymore. The full log is in the [bug itself](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1026625).

Overall, I get the feeling that this package is not very actively maintained nowadays. Am I right?

Thanks for this code, and for your time.

**Comment by ppalaga** [comment]
Yeah, I have not touched this code since 2015. Making the build pass would require switching to public SSL APIs and generally make it compile with newer JDK versions. TBH, I do not see myself having any time for that. Would anybody be ready to take over this project?

**Comment by ebourg** [comment]
Actually the version 2.0.0.Alpha3 build fine with Java 21. The build failure in Debian can be fixed by upgrading the package to the latest version.
